packages/flutter_tools/lib/src/features.dart

```diff
@@ -88,7 +88,7 @@ const Feature flutterWebFeature = Feature(
 
 /// The [Feature] for macOS desktop.
 const Feature flutterMacOSDesktopFeature = Feature(
-  name: 'Flutter Desktop for macOS',
+  name: 'Flutter for macOS desktop',
```

Comment:
`"Flutter for macOS desktop"` ==> `"Flutter for desktop on macOS"`? `"Flutter for desktop (macOS)"`?

The changes above would let the name read `Flutter for desktop` (which would parallel the `Flutter for web` name).

lgtm either way

Reply:
Done